src/lib/components/list-item.svelte

```
@@ -89,7 +107,7 @@
 {/snippet}
 
 {#if mediaURL}
-    <!-- svelte-ignore svelte/no-navigation-without-resolve -->
+    <!-- svelte-ignore a11y_no_static_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)